### PR TITLE
[PRM-924] Added url for vat-through-software and logic for agent version

### PR DESCRIPTION
**Pull request by JBlacklaws**

### CODE REVIEW

**Review by ruthdas03 — CHANGES_REQUESTED** [review]
Please add  test case in IndexViewSpec for this change.
Also we need a PT for the same.

File: conf/application.conf [hunk 1]
```diff
@@ -86,6 +86,7 @@ microservice {
 
 urls {
   vatOverview = "http://localhost:9152/vat-through-software/vat-overview"
+  vatAgentClientLookUp = "http://localhost:9149/vat-through-software/representative/"
```
[the remark this answers is not in the document]
Comment (JBlacklaws) [reply conf/application.conf:89]: I assumed the client-vat-number was supposed to be whatever client VRN is?

[the remark this answers is not in the document]
Comment (LewisMitchell) [reply conf/application.conf:89]: This was a mistake on my side the URL should be: http://localhost:9149/vat-through-software/representative/client-vat-account - I'll update the ticket

[the remark this answers is not in the document]
Comment (JBlacklaws) [reply conf/application.conf:89]: I'll update the url here

**Review by ruthdas03 — APPROVED** [review]

**Pull request merged**